pages.de/windows/cd.md

```diff
@@ -0,0 +1,20 @@
+# cd
+
+> Zeigt den Namen des aktuellen Arbeitsverzeichnisses an oder ändert diesen.
```

Comment:
Ich würde lieber "[...] oder ändert dieses." schreiben, sonst könnte man meinen der Name des aktuellen Arbeitsverzeichnisses würde geändert werden.

Reply:
I somehow managed to delete the original branch for this PR. So I created a new one with the fix you mentioned here. #4575